app/Models/Organisation.php

```diff
@@ -320,14 +321,43 @@ public function sector()
         );
     }
 
+    public function sroOfficer(): HasOne
+    {
+        return $this->hasOne(
+            User::class,
+        )->where('is_sro', 1);
+    }
+
+
+    public function scopeGetOrganisationsProjects($query)
+    {
+        $results = array();
+        $organisations = $query->get();
+
+        foreach ($organisations as $organisation) {
+            if (count($organisation->projects)) {
```

Comment:
Might be able to optimise this to simply `if ($organisation->projects)` to avoid counting?